docs/notebooks/langmuir_samples/_generate_noisy.ipynb

```diff
@@ -0,0 +1,171 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Generate noisy signals\n",
+    "\n",
+    "The default Langmuir samples are not noisy and, thus, do not properly reflect real world data.  This notebook will add noise to the signals to help simulate real world data."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import astropy.units as u\n",
+    "import matplotlib.pyplot as plt\n",
+    "import numpy as np\n",
+    "import os\n",
+    "\n",
+    "from plasmapy.utils.decorators import validate_quantities\n",
+    "\n",
+    "plt.rcParams[\"figure.figsize\"] = [10.5, 0.56 * 10.5]"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# noise generator\n",
+    "@validate_quantities\n",
+    "def add_noise(signal, snr_db: u.dB = 10 * u.dB):\n",
```

Comment:
Logarithmic units! 💯   Oh wait, I meant 2️⃣.

Reply:
Somewhat more seriously, I wonder if we should have functions in PlasmaPy that can be used to add different kinds of noise.  That could potentially go in utils.  No need to do anything about that now...perhaps we could wait until later when we have need for something like this again.

Reply:
I was thinking the function might be useful in plasmapy, specifically in `plasmapy.analysis.signal` (similar in scope to `scipy.signal`).  I wasn't sure though, so I just left the notebook where it is for now.  The notebook isn't actually exposed anywhere in the documentation (on purpose).  It's only being used to make a Langmuir sweep with noise, which is then used in the Langmuir notebooks.  I just left the notebook in place for record purposes.